.circleci/config.yml

```diff
@@ -1,6 +1,30 @@
 version: 2.1
 
 jobs:
+  lint:
+    docker:
+      - image: circleci/golang:1.16
+    steps: # steps that comprise the `build` job
+      - checkout # check out source code to working directory
+      - run: "echo $(($(date +%s)/604800)) > current_week"
+      - restore_cache:
+          key: golang-lint-ci-{{ checksum "current_week" }}
+      - run:
+          name: install lint
+          command: |
+            if [[ ! -e /usr/local/bin/golangci-lint ]]; then
+              cd /usr/local
```

Comment:
not sure what's causing the permissions issue that's failing in CI. Is there are a reason the docker image for this job is `circleci/golang:1.16`. In most of our CI we use `ubuntu-2004:202101-01`

Reply:
Looks like there is a golangci docker image you could use https://hub.docker.com/r/golangci/golangci-lint

Reply:
:+1: thanks! I switched to that image and removed the install step, which cleaned things up a lot